cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -66,10 +67,8 @@ case class InputParameterCommandPart(commandInputParameter: CommandInputParamete
       val womValue: WomValue = commandInputParameter match {
         case cip: CommandInputParameter =>
           val localizedId = LocalName(FullyQualifiedName(cip.id).id)
-          inputsMap.get(localizedId) match {
-            case Some(x) => valueMapper(x)
-            case _ => throw new RuntimeException(s"could not find $localizedId in map $inputsMap")
-          }
+          inputsMap.get(localizedId).cata(
```

Comment:
... and what's a `cata`?

Reply:
That's how its close friends refer to a [catamorphism](https://en.wikipedia.org/wiki/Catamorphism).